Fix part of #3950  - replace jinja templates

adarsh1021 opened the pull request:
<!--
  - Thanks for submitting code to Oppia! Please fill out the following as part of
  - your pull request so we can review your code more easily.
  -->

## Explanation
Fixes part of #3950 

Updated jinja if statements to ng-if format. 
Files changed - 
`header_css_libs.html`
`header_js_libs.html`
<!--
  - Explain what your PR does. If this PR fixes an existing bug, please include 
  - "Fixes #bugnum:" in the explanation so that GitHub can auto-close the issue 
  - when this PR is merged.
  -->

adarsh1021 committed:
updated jinja in header_css_libs.html

adarsh1021 committed:
updated jinja in header_js_libs.html

apb7 commented:
@adarsh1021: The end-to-end tests fail during the Travis build. Please test the changes locally before pushing them, in general.  
Thanks!

jervis446 commented:
Hi,@adarsh1021  Thanks for your PR,Travis is failing Please take a look.

adarsh1021 committed:
updated header_js_libs.html

adarsh1021 committed:
Merge branch 'develop' of https://github.com/oppia/oppia into replace-jinja

adarsh1021 commented:
Hello !
This is my first pull request. 
Well the issue seems to be that travis jobs are exceeding the time limit and so they are terminated. 

I replaced the jinja templates in both `header_css_libs.html` and `header_js_libs.html` in my first commit. But then in `header_js_libs.html` the ng-if statement was added in a `<script>` tag, so it was not taking affect, as scripts are loaded by the browser before Angular manipulates the DOM. So the workaround would be to load the script manually using a function in angualr. 
Anyways, so I changed the code back to how it was in `header_js_libs.html`. 
But still travis is not checking 

It would be great if you could help me with this issue, and suggest what I should do about it. 

Thank you :)

seanlip commented:
Did you test this with a local dev server? I suspect you'd get console errors locally as well, it's not just a Travis problem.

Figuring out how to handle this sort of issue can be a bit tricky. The aim is to remove the Jinja templates, but keep the site working. However, I'm not sure what the solution is offhand, and this will require a bit of debugging and experimentation. If it's too hard, it's fine to take up a different issue!

pranavsid98 commented:
Hi @adarsh1021 Any updates on this?

chari1239 commented:
I worked on this(I changed the `header_css_libs.html  , header_js_libs.html` files.It's working.This is my first time making a pull request. Help me how to contribute. @apb7

chari1239 commented:
I changed the files `header_css_libs.html header_js_libs.html` . Its working in my dev environment without any error in console.How can I make a pull request. @apb7

apb7 commented:
Hi @chari1239, this PR has been assigned to @adarsh1021. Please feel free to pickup any other file from Issue #3950. Thanks!

chari1239 commented:
@apb7  There was  no response from @adarsh1021 for a long time.So I thought he is not working on this issue. 
I will take other files. I also worked on other issue #4819 ,please check it also once. Thanks.

1995YogeshSharma commented:
Hi @adarsh1021 , are you working on this?

shubha1593 commented:
Closing this PR since due to lack of response.
